config/initializers/geocoder.rb

```diff
@@ -0,0 +1,6 @@
+Geocoder.configure(
+  timeout: 3,
+  lookup: :google,
+  api_key: ENV['GEOCODER_APIKEY'],
```

Comment:
Where is this ENV variable set?

Reply:
In a hidden .env file

Reply:
Where? I see no dotenv gem

Reply:
Yeah it wasn't added.. my bad :) Up and running now